### ISSUE TITLE
Suggestion: increase the timeout for the turns of TB battle

### ISSUE THREAD

**Issue by souliane** [MrStahlfelge/lightblocks#29]
The TB battle's timeout is quite short. Some days you have no time to play and then you feel bad for having let your opponents play their turns for nothing... would it be possible to extend the timeout to 48 or even 72 hours?

**Comment by MrStahlfelge** [comment]
I've been thinking about the suggestion the last days. There is no technical reason for the restriction, it was a design decision based on how I have seen other games handle this. I miss my battles from time to time myself and know about the problem. I think best approach is to enhance the automatic timeout to 72 hours, but the waiting player can withdraw after waiting for 24 hours.

**Comment by hzimm** [comment]
Hello,

I'll back the request as the game is fantastic and the multi unique - it brings my brother and I together for a few minutes of play every week while we are at opposite sides of the planet. But the huge difference in the timezones etc. makes it hard to follow and we end up forfeiting every second game.

I supported the development and thank you for your great work.

**Comment by MrStahlfelge** [comment]
The behaviour is now changed. Open battles are closed after 24 hours as it already was. Running battles are closed after 72 hours, so this is far more time to play.
Let me know if it is not working as intended.

**Comment by souliane** [comment]
Awesome, thanks! I haven't "used" it yet but I will tell you if I encounter some issues.
